CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [12.4.0](https://github.com/sbb-design-systems/sbb-angular/compare/12.3.0...12.4.0) (2021-09-02)
```

Comment:
Changelog will be automatically generated based on merged PRs (or other commits on master).
Please do not change it.